src/serialize/other.rs

```diff
@@ -0,0 +1,41 @@
+//! Message sent from datapath to CCP when a new flow starts.
```

Comment:
why is this called "other" if it's actually a "DatapathInitMsg" or whatever

Reply:
whoops, looks like i accidentally copied that comment, it's not correct :)

other is just a wrapper around RawMsg that owns the bytes to remove the lifetime specifier